Makefile

```diff
@@ -27,6 +27,8 @@ proto:
 
 clean:
 	for d in $(SUBDIRS); do $(MAKE) -C $$d clean; done
+	# --force is used to squash errors that pertain to file not existing
+	rm --force sandbox-test-cri-build-stamp
```

Comment:
Instead of using `--force`, prefix the command with `-` so it becomes `-rm`.  See the [`make` manual](https://www.gnu.org/software/make/manual/html_node/Errors.html) for details.